libgearman-server/plugins/queue/sqlite/instance.cc

```diff
@@ -432,6 +432,9 @@ gearmand_error_t Instance::done(gearman_server_st*,
                                    const char *function_name,
                                    size_t function_name_size)
 {
+  if(is_store_on_shutdown())
```

Comment:
Please add a space between "if" and the open parathensis on this line. Thank you.

Reply:
sure